### PR TITLE
fix: wrong private schedule assignment in default popup (resolve #927)

### LINKED ISSUE #927: Class: private / public, in the raw element of the schedule does not get updated.
When I create the schedule I can click on the lock icon which adds "class: private" in the "raw" element of scheduleData. However, when I try to update the same schedule afterwards - nothing happens. The "changes" object in the beforeUpdateSchedule function is null if I don't change something else in the pop-up, apart from just clicking the icon. 

What I'm ultimately trying to accomplish is to have a schedule event which the user won't be able to drag (like the read-only schedules), but will be able to edit (unlike the read-only schedules), and after editing if they change this property - it becomes draggable again. I'm considering using the lock icon for this. Please let me know If there's a better way to accomplish something similar.

Thanks

### PR DESCRIPTION
**Pull request by adhrinae**
### Please check if the PR fulfills these requirements
- [x] It's submitted to right branch according to our branching model
- [x] It's right issue type on title
- [x] When resolving a specific issue, it's referenced in the PR's title (e.g. `fix #xxx[,#xxx]`, where "xxx" is the issue number)
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes/features)
- [x] Docs have been added/updated (for bug fixes/features)
- [ ] It does not introduce a breaking change or has description for the breaking change

### Description

- Changed wrong JSDoc comment (#930)
- Fixed wrong usage of the `raw` property in the `Schedule` model. (#927)
  - The `raw` property is only for library consumers who want to save arbitrary data in the `Schedule` instance.
  - There is already `isPrivate` in the `Schedule` model. so replace logic to use it.
  - Also example code has been changed not to use `raw` properties inappropriately.

---
Thank you for your contribution to TOAST UI product. 🎉 😘 ✨

### CODE REVIEW

**Review by jwlee1108 — APPROVED** [review]
리뷰 완료합니다.

File: test/app/view/scheduleCreationPopup.spec.js [hunk 1]
```diff
@@ -91,6 +91,93 @@ describe('ScheduleCreationPopup date range picker', function() {
...
+        var spy = jasmine.createSpy('beforeCreateSchedule');
+        popup.on('beforeCreateSchedule', spy);
+        popup.render(mockViewModel);
+
+        togglePrivateButton();
+        clickSaveButton();
+
+        result = spy.calls.argsFor(0)[0];
+
+        expect(result.isPrivate).toBe(true);
```
Comment (lja1018) [inline test/app/view/scheduleCreationPopup.spec.js:138]:
```suggestion
        result = spy.calls.argsFor(0)[0].isPrivate;

        expect(result).toBe(true);
```
는 어떠신가요?

File: test/app/view/scheduleCreationPopup.spec.js [hunk 2]
```diff
@@ -91,6 +91,93 @@ describe('ScheduleCreationPopup date range picker', function() {
...
+                title: 'mock schedule',
+                start: new TZDate('2015-05-01T09:00:00'),
+                end: new TZDate('2015-05-01T10:00:00'),
+                guide: jasmine.createSpyObj('guide', ['clearGuideElement']),
+                isPrivate: true
+            }
+        };
+        popup.render(mockEditModeViewModel);
+
+        expect(getPrivateButton().classList.contains('tui-full-calendar-public')).toBe(false);
```
Comment (lja1018) [inline test/app/view/scheduleCreationPopup.spec.js:177]: `getPrivateButton().classList.contains('tui-full-calendar-public')` 변수화해서 넣는건 어떠신가요?

**Review by lja1018 — COMMENTED** [review]
리뷰완료합니다.
빌드에러 확인 부탁드릴게요!

File: test/app/view/scheduleCreationPopup.spec.js (hunk 1 above)
Comment (adhrinae) [reply test/app/view/scheduleCreationPopup.spec.js:138]: 스파이에서 호출된거에 바로 프로퍼티를 조회하는건 별로 내키진 않는데 `result` 라는 변수명이 좀 더 의미있게 바꾸어야겠다는 힌트는 얻었네요.

File: test/app/view/scheduleCreationPopup.spec.js (hunk 2 above)
Comment (adhrinae) [reply test/app/view/scheduleCreationPopup.spec.js:177]: 👌

File: test/app/view/scheduleCreationPopup.spec.js [hunk 3]
```diff
@@ -91,6 +91,95 @@ describe('ScheduleCreationPopup date range picker', function() {
...
+                start: new TZDate('2015-05-01T09:00:00'),
+                end: new TZDate('2015-05-01T10:00:00'),
+                guide: jasmine.createSpyObj('guide', ['clearGuideElement']),
+                isPrivate: true
+            }
+        };
+        popup.render(mockEditModeViewModel);
+
+        isPrivateEnabled = !getPrivateButton().classList.contains('tui-full-calendar-public');
+        expect(isPrivateEnabled).toBe(true);
```
Comment (js87zz) [inline test/app/view/scheduleCreationPopup.spec.js:179]: v1에서는 jasmine이라 `hasClass` 같은 매처는 쓸 수 없나요?

**Review by js87zz — APPROVED** [review]
리뷰 완료입니다!

File: test/app/view/scheduleCreationPopup.spec.js (hunk 3 above)
Comment (adhrinae) [reply test/app/view/scheduleCreationPopup.spec.js:179]: 다른 테스트 찾아보니까 쓰는 부분이 있네요. 수정해둘게요.

**Pull request merged**